### PR TITLE
Rename apps pack install button

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
More precise wording about the actions which are triggered by the button.

I cannot test from my side ... can a reviewer do this ? 
Also, the wording isn't located in the usual syntax which is "t('project', 'text')" ... so I'm not sure if it works correctly ?!

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/AppsManagementContext.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ public static function disableButtonForAnyApp() {
 	 * @return Locator
 	 */
 	public static function enableAllBundleButton($bundle) {
-		return Locator::forThe()->xpath("//div[@class='apps-header']/h2[normalize-space() = '$bundle']/input[@value='Enable all']")->
+		return Locator::forThe()->xpath("//div[@class='apps-header']/h2[normalize-space() = '$bundle']/input[@value='Download and enable all']")->
```
Comment (ChristophWurst) [inline tests/acceptance/features/bootstrap/AppsManagementContext.php:89]: this is an integration test, not the front-end source

Comment (kesselb) [reply tests/acceptance/features/bootstrap/AppsManagementContext.php:89]: Likely https://github.com/nextcloud/server/blob/c1975a634cb6d40d9024c7b329c1a5bb0d01aa24/apps/settings/src/components/AppList.vue#L213

Comment (Jerome-Herbinet) [reply tests/acceptance/features/bootstrap/AppsManagementContext.php]: OK thanks, it's done and I reverted the first commit.

Comment (ChristophWurst) [reply tests/acceptance/features/bootstrap/AppsManagementContext.php]: There's a chance you need to rename at both locations so the automated tests still pass. Continuous integration tests will tell :popcorn:

Comment (Jerome-Herbinet) [reply tests/acceptance/features/bootstrap/AppsManagementContext.php:89]: Done @ChristophWurst - Wait and see.

**Review by jancborchardt — APPROVED** [review]
Nice wording fix!

**Comment by Jerome-Herbinet** [comment]
Oups, messed up. 
Please go to the respawn : https://github.com/nextcloud/server/pull/39754

**Pull request closed without merging**